index.js

```diff
@@ -220,36 +246,6 @@ module.exports = {
     return utils.unique(locales);
   },
 
-  findIntlAddons() {
-    let projectName = this.app.project.name();
-    let addons = this.app.project.addons;
-    let registered = new Set();
-
-    let find = function(list, addon) {
-      // Only handle each addon once
-      if (registered.has(addon.name)) {
-        return list;
-      }
-
-      let translationPath = addon.pkg['ember-addon'].translationPath || 'translations';
```

Comment:
I'm dropping `translationPath` in favor of `treeForTranslations`.  Addons can return a `this.treeGenerator(<any path>)`